frontend/src/lib/components/PropertyFilters/PropertyFilter.js

```diff
@@ -20,6 +20,7 @@ export function PropertyFilter({ index, onComplete, logic }) {
                     filterOption={(input, option) => option.children?.toLowerCase().indexOf(input.toLowerCase()) >= 0}
                     onChange={(_, new_key) => setFilter(index, new_key.children, undefined, operator, new_key.type)}
                     style={{ width: '100%' }}
+                    virtual={false}
```

Comment:
technically "reduces" list performance because it won't smart render components. disabled for the sake of tests because the smart rendering was causing unpredictable behavior so the test couldn't properly find the list option to select. We're not loading an absurd amount of properties/infinite list so it shouldn't be impactful